### DISCUSSION TITLE
Neovim is not being configured by NvChad

### DISCUSSION THREAD

**Discussion by Penqoin** [NvChad/NvChad#2405] [Q&A]
I've tried to apply NvChad to Neovim config, but it doesn't apply when I start nvim. There are no errors, :checkhealth shows no problems. I've tried a clean install, downgrading to multiple versions and edited the lua.init with no luck.

I am using arch and have not done any changes to the default nvim or the NvChad config.

What's the issue?

**Comment by siduck** [comment]
make a video and show me

**Reply by Penqoin** [reply]
I did a clean install
https://github.com/NvChad/NvChad/assets/90331303/4a6aeef7-91d5-4a2d-8386-19cd5b960f4f

**Reply by siduck** [reply]
@TheOnlyTater show me output of ls .config/nvim and ls .local/share/nvim

**Reply by Penqoin** [reply]
.config/nvim 
init.lua  LICENSE  lua/

.local/share/nvim is empty

**Reply by siduck** [reply]
rm -rf ~/.local/share/nvim && rm -rf ~/.config/nvim, try now

and make sure you have latest version of git

**Reply by Penqoin** [reply]
I have the latest version of git and deleting the directories did not fix it

**Reply by siduck** [reply]
this is the weirdest issue we've faced so far man. idek whats causing it, but i'm sure lazy isnt able to do bootstrapping. Ask on our discord server, someone might help you in my absence
